test/integration/services/data/test-get-all-open-reductions.js

```diff
@@ -0,0 +1,35 @@
+const expect = require('chai').expect
+
+const appWorkloadOwnerHelper = require('../../../helpers/data/app-workload-owner-helper')
+const appReductionsHelper = require('../../../helpers/data/app-reductions-helper')
+
+const getAllOpenReductions = require('../../../../app/services/data/get-all-open-reductions')
+
+var inserts = []
+
+describe('services/data/get-all-open-reductions', function () {
+  before(function (done) {
+    appWorkloadOwnerHelper.insertDependencies(inserts)
+      .then(function (builtInserts) {
+        return appReductionsHelper.insertDependencies(inserts)
+        .then(function (builtInserts) {
+          inserts = builtInserts
+          done()
+        })
+      })
+  })
+
+  it('should retrieve the open reductions in system', function (done) {
+    getAllOpenReductions().then(function (results) {
+      results.forEach(function (result) {
+        expect(['ACTIVE', 'SCHEDULED', null]).to.include(result.status)
+      })
+      done()
+    })
+  })
```

Comment:
Run a knex query to pull back all the open reductions and verify your results is the same set?

Reply:
Change to do something similar, get all reductions, then verify that the ones that the original query didn't bring back are what I expect